pyproject.toml

```diff
@@ -1,5 +1,45 @@
+[build-system]
+requires = ["setuptools >=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "keras"
+authors = [
+    {name = "Keras team", email = "keras-users@googlegroups.com"},
+]
+description = "Multi-backend Keras"
+readme = "README.md"
+requires-python = ">=3.9"
+license = {text = "Apache License 2.0"}
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3 :: Only",
+    "Operating System :: Unix",
+    "Operating System :: MacOS",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Software Development",
+]
+dynamic = ["version", "dependencies"]
+
+[project.urls]
+Home = "https://keras.io/"
+Repository = "https://github.com/keras-team/keras"
+
+[tool.setuptools.dynamic]
+version = {attr = "keras.src.version.__version__"}
+dependencies = {file = "requirements-common.txt"}
+
+[tool.setuptools.packages.find]
+include = ["keras", "keras.*"]
```

Comment:
No excludes?

Reply:
setup.py had `exclude=("*_test.py", "benchmarks")`, but this didn't do anything, so was dropped with the transition. The reason why it didn't do anything is that only Python modules under "keras" are found (thus "benchmarks" was already excluded), and individual Python files cannot be excluded -- only modules can be excluded. But I see that pip_build.py has logic to ignores the "_test.py" files.